information/specification.md

```diff
@@ -0,0 +1,706 @@
+outputs from the UTxO cannot be subdivided, and must be spent completely in a
+given transaction.  Consequently, in the case where it's not possible to cover
+the target amount exactly, the wallet software must arrange that change is paid
+back to the wallet.
+
+Coin selection refers to the process of choosing _unspent outputs_ from a
+wallet's UTxO set in order to make one or more payments, and arranging for
+_change_ to be paid back to the wallet by creating one or more _change
+outputs_.
+
+## Why is Coin Selection Non-Trivial?
+
+### Issues
+
+There are a number of issues which make the problem of coin selection more
+complicated than it would initially appear.
+
+ * Each transaction has a _maximum size_, as defined by the protocol. The size
+   of a transaction increases as we add more inputs or outputs.
+
+   Therefore, there's a practical limit on the number of coins we can select
+   for any given transaction.
+
+ * The most obvious strategy for coin selection, which consists of trying to
+   get as close to the requested value as possible, will tend (over time) to
+   create a lot of _dust_: small unspent outputs.
+
+   Dust outputs are a problem, because even if the total value of dust in a
+   wallet is more than enough to cover a given target amount, if we attempt to
+   include that dust in a given transaction, we may run out of space (by
+   reaching the transaction size limit) before we can cover the target amount.
+
+ * The most obvious strategy for paying change, which consists of making a
+   single change output with the exact excess value, will tend (over time) to
+   reduce the size of the UTxO set. This is bad for two reasons:
+
+    1. Having a small UTxO set limits the number of payments that we can make
+       in parallel. Since a single UTxO entry can only be used to pay for a
+       single output, we need at least as many UTxO entries as there are
+       outputs.
+
+    2. The approach of coalescing all change into a single output is widely
+       considered to have negative privacy implications. The discussion of
+       privacy issues is beyond the scope of this article.
+
+### Goals
+
+There are several desirable properties that we would like a coin selection
+algorithm to have. These properties include:
+
+ * A coin selection algorithm should, over the course of time, aim to generate
+   and maintain a UTxO set with _useful_ outputs: that is, outputs that allow
+   us to process future payments with a _minimum_ number of inputs.
+
+ * A coin selection algorithm should employ strategies to limit the
+   amount of _dust_ that accumulates in the UTxO set.
+
+For more information on dust avoidance, see [Self Organisation in Coin
+Selection](#self-organization-in-coin-selection).
+
+# Definitions
+
+### Address
+
+An _address_ is a unique identifier that represents a destination for a payment.
+
+Addresses are owned (and generated) by individual wallets.
```

Comment:
*are typically owned 

In practice, HD wallets aren't necessary to generate addresses and interact with the blockchain.

Reply:
> *are typically owned
> 
> In practice, HD wallets aren't necessary to generate addresses and interact with the blockchain.

Good point!

Fixed in 786e107d21138b5ab9797f486ac2bc96f159f405.